srcpkgs/mdadm/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'mdadm'
 pkgname=mdadm
 version=4.0
```

Comment:
maintainer should be

Leah Neukirchen <leah@vuxu.org>

license is GPL-2.0-or-later